textworld/envs/wrappers/filter.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+
+from typing import Tuple, Mapping, Any, List, Iterable
+
+from textworld.core import GameState, Wrapper
+
+
+class EnvInfos:
+    """
+    Customizing what information will be returned by an environment.
+
+    Information can be requested by setting one or more attributes to True.
+    The attribute `extras` should be a list of strings corresponding to
+    information specific to certain games.
+
+    """
+
+    __slots__ = ['description', 'inventory', 'location',
+                 'facts',
+                 'max_score', 'objective',
+                 'entities', 'verbs', 'command_templates',
+                 'admissible_commands', 'intermediate_reward',
+                 'policy_commands',
+                 'extras']
+
+    def __init__(self, **kwargs):
+        #: bool: Text description of the current room, i.e. output of the
+        #:       `look` command.
+        #:       This information changes from one step to another.
+        self.description = kwargs.get("description")
```

Comment:
Might be nice to do `.get("description", False)` if the type is documented as `bool`

Reply:
Done